sulis_module/src/item.rs

```diff
@@ -132,15 +133,24 @@ impl Item {
             &all_adjectives,
         );
 
+        let mut unidentified_name = None;
+
         let mut name = String::new();
         for adj in all_adjectives.iter() {
             if let Some(ref prefix) = adj.name_prefix {
+                if unidentified_name.is_none() &&
+                    "Exceptional Fine Masterwork Rusty Worn ".find(prefix).is_none() {
```

Comment:
The data driven values from these adjectives shouldn't be hardcoded here.  The "Unidentified" string shouldn't either.

We should use the `unidentified_item_adjective` from the `Rules`.  We should just be able to set a name_prefix for that adjective and append it to the base item name.